### ISSUE TITLE
get-content via drive-provider appears to enter infinite loop

### ISSUE THREAD

**Issue by NeilMacMullen** [NeilMacMullen/jumpfs#10]
Applies to 1.2.0

You can get the path of a bookmark via `$jfs:x` but using `get-content jfs:x` results in an infinite loop.

Debug output....

```
get-content jfs:j
DEBUG: MakePath:  parent:'' child:'j'
DEBUG: MakePath:  parent:'jfs:\' child:'j'
DEBUG: GetChildName:
DEBUG: ItemExists:  jfs:\j
DEBUG: GetParentPath:
DEBUG: GetParentPath:
DEBUG: MakePath:  parent:'jfs:\' child:'j'
DEBUG: MakePath:  parent:'' child:'j'
DEBUG: MakePath:  parent:'jfs:\' child:'j'
DEBUG: GetChildName:
DEBUG: ItemExists:  jfs:\j
DEBUG: GetParentPath:
DEBUG: GetParentPath:
DEBUG: MakePath:  parent:'' child:'j'
DEBUG: MakePath:  parent:'jfs:\' child:'j'
DEBUG: GetChildName:
DEBUG: ItemExists:  jfs:\j
DEBUG: GetParentPath:
DEBUG: GetParentPath:
DEBUG: MakePath:  parent:'jfs:\' child:'j'
DEBUG: GetChildName:
```

**Comment by NeilMacMullen** [comment]
Looks like GetParentPath needs to be implemented.

**Comment by NeilMacMullen** [comment]
Fixed in 1.3
